docs/source/includes/elasticluster_setup.rst

```diff
@@ -0,0 +1,358 @@
+.. _S3IT: http://www.s3it.uzh.ch/
+.. _Elasticluster: https://elasticluster.readthedocs.org
+.. _virtualenv: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+.. _gcloud: https://cloud.google.com/sdk/
```

Comment:
Will updates to conf.py only get picked up when the content is staged to
readthedocs?

On Mon, Jun 29, 2015 at 5:55 PM Nicole Deflaux <notifications@github.com>
wrote:

> In docs/source/includes/elasticluster_setup.rst
> <https://github.com/googlegenomics/start-here/pull/59#discussion_r33531397>
> :
>
> > @@ -0,0 +1,358 @@
> > +.. _S3IT: http://www.s3it.uzh.ch/
> > +.. _Elasticluster: https://elasticluster.readthedocs.org
> > +.. _virtualenv: http://docs.python-guide.org/en/latest/dev/virtualenvs/
> > +.. _gcloud: https://cloud.google.com/sdk/
>
> Consider re-using or adding to the common links here:
> https://github.com/googlegenomics/start-here/blob/master/docs/source/conf.py#L139
> )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/start-here/pull/59/files#r33531397>.
>